### PR TITLE
Implemented addmv(),addcmul(),addcdiv(),addbmm(),baddmm() and rectified addmm()

### PR DESCRIPTION
**Pull request by bartimaeus12**
fixes issues
 #21 
#22 
#23 
#25 
#26 
#142

### CODE REVIEW

**Comment by erfannoury** [comment]
This PR has lots of `pep8` issues. Can you please fix them? You can use tools like `flake8` to find the issues.

**Comment by bartimaeus12** [comment]
pep8 issues fixed @erfannoury

**Comment by erfannoury** [comment]
@bartimaeus12 thank you.

File: tests/test_tensor.py [hunk 1]
```diff
@@ -72,89 +67,202 @@ def testInplace(self):
...
 
     def testIneqOp(self):
-        t1 = TensorBase(np.array([1, 2, 3]))
-        t2 = TensorBase(np.array([1, 4, 5]))
+        t1 = TensorBase(np.array([1,  2,  3]))
+        t2 = TensorBase(np.array([1,  4,  5]))
         self.assertTrue(t1 != t2)
 
 
-class IndexTests(unittest.TestCase):
```
Comment (erfannoury) [inline tests/test_tensor.py]: Why are these tests being removed?

Comment (bartimaeus12) [inline tests/test_tensor.py]: I think i did it by mistake i am closing this PR and opening another one

**Pull request closed without merging**